src/scss/components/_card.scss

```diff
@@ -1,13 +1,13 @@
 .card {
   background-color: $white;
+  border: 0;
+  margin: 0 auto auto auto;
   padding: 2em;
-  position: relative;
-  margin: auto;
   width: 100%;
-  box-shadow: 0 0.25em 0.25em 0 rgba(210, 210, 210, 0.5);
-  border-radius: 0.125em;
 
   @media ( min-width: 25em ) {
```

Comment:
You might want to make this breakpoint a little wider: when choosing "Pixel 2 XL" in the responsive design mode (it has a width of 411px) we still get the grey background.

Reply:
Done. I set it to 36em (576px), which is what bootstrap uses as a cutoff.